docs/mmteb/points/585.jsonl

```diff
@@ -0,0 +1,3 @@
+{"GitHub": "dokato", "New dataset": 6}
```

Comment:
Sorry it's gonna be only 2 since we already handle Finnish here: https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/Classification/multilingual/MassiveScenarioClassification.py